compiler/luci/import/src/Importer.test.cpp

```diff
@@ -16,13 +16,289 @@
+  }
+
+  uint32_t add_subgraph()
+  {
+    model->subgraphs.emplace_back(new circle::SubGraphT);
+    model->subgraphs.back()->name = "";
+    model->subgraphs.back()->data_format = circle::DataFormat_CHANNELS_LAST;
+    return model->subgraphs.size() - 1;
+  }
+
+  void add_subgraph_inputs(uint32_t subgraph_id, const std::vector<uint32_t> &inputs)
+  {
+    model->subgraphs[subgraph_id]->inputs.assign(inputs.begin(), inputs.end());
+  }
+
+  void add_subgraph_outputs(uint32_t subgraph_id, const std::vector<uint32_t> &outputs)
+  {
+    model->subgraphs[subgraph_id]->outputs.assign(outputs.begin(), outputs.end());
+  }
+
+  uint32_t add_builtin_opcode(circle::BuiltinOperator opcode)
+  {
+    uint32_t id = model->operator_codes.size();
+    model->operator_codes.emplace_back(new circle::OperatorCodeT);
+    model->operator_codes[id]->builtin_code = opcode;
+    model->operator_codes[id]->version = 1;
+    return id;
+  }
+
+  uint32_t add_buffer()
+  {
+    model->buffers.emplace_back(new circle::BufferT);
+    return model->buffers.size() - 1;
+  }
+
+  uint32_t add_float_tensor(uint32_t graph_id, const std::vector<int32_t> &shape,
+                            uint32_t buffer_id)
+  {
+    auto &graph = model->subgraphs[graph_id];
+    uint32_t idx = graph->tensors.size();
+    graph->tensors.emplace_back(new circle::TensorT);
+    graph->tensors[idx]->shape = shape;
+    graph->tensors[idx]->type = circle::TensorType_FLOAT32;
+    graph->tensors[idx]->buffer = buffer_id;
+    graph->tensors[idx]->name = std::to_string(idx);
+    graph->tensors[idx]->quantization.reset(nullptr);
+    graph->tensors[idx]->is_variable = false;
+    graph->tensors[idx]->sparsity.reset(nullptr);
+    (void)graph->tensors[idx]->shape_signature;
+    return idx;
+  }
+
+  uint32_t add_builtin_operator(uint32_t graph_id, uint32_t opcode_id,
+                                const std::vector<uint32_t> &inputs,
+                                const std::vector<uint32_t> &outputs)
+  {
+    auto &graph = model->subgraphs[graph_id];
+    auto idx = graph->operators.size();
+    graph->operators.emplace_back(new circle::OperatorT);
+    graph->operators[idx]->opcode_index = opcode_id;
+    graph->operators[idx]->inputs.assign(inputs.begin(), inputs.end());
+    graph->operators[idx]->outputs.assign(outputs.begin(), outputs.end());
+    graph->operators[idx]->builtin_options.Reset();
+    (void)graph->operators[idx]->custom_options;
+    graph->operators[idx]->custom_options_format = circle::CustomOptionsFormat_FLEXBUFFERS;
+    (void)graph->operators[idx]->mutating_variable_inputs;
+    (void)graph->operators[idx]->intermediates;
+    return idx;
+  }
+
+  uint32_t add_plan_metadata(uint32_t buffer_id)
+  {
+    uint32_t idx = model->metadata.size();
+    model->metadata.emplace_back(new circle::MetadataT);
+    model->metadata[idx]->name = "ONE_execution_plan_table";
+    model->metadata[idx]->buffer = buffer_id;
+    model->buffers[buffer_id]->data.resize(4);
```

Comment:
(optional) How about set counter as 0 explicitly to increase code readability? 

```suggestion
    model->buffers[buffer_id]->data.resize(4);
    uint32_t &counter = *reinterpret_cast<uint32_t *>(model->buffers[buffer_id]->data.data());
    counter = 0;
```